sdk/servicebus/azure-servicebus/samples/async_samples/send_topic_async.py

```diff
@@ -21,20 +21,25 @@
 
 
 async def send_single_message(sender):
-    message = Message("DATA" * 64)
-    await sender.send_messages(message)
+    message = Message("Single Message")
+    await sender.send(message)
```

Comment:
as above (re: send_messages; this is a bunch of places so probably just do a find/replace on .send( and the other variants)

Reply:
updated!